database/init_db.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+psql -U postgres -d postgres <<-EOSQL
+    CREATE DATABASE ukrdc4;
+    CREATE USER ukrdc WITH LOGIN PASSWORD 'password123';
+EOSQL
```

Comment:
If you do...

`ALTER DATABASE "ukrdc4" SET search_path TO "$user",extract;`

That will sort the issue with search paths you're having somewhere else in this.